stdlib/experimental/dynamic/dynamic.flux

```diff
@@ -0,0 +1,17 @@
+// Package dynamic provides tools for working with values of unknown types.
+//
+// ## Metadata
+// introduced: NEXT
+//
+package dynamic
+
+
+// dynamic wraps a value so it can be used as a `dynamic` value.
+//
+// ## Parameters
+// - v: the value to wrap
+//
+// ## Examples
+//
+// TODO(onelson): need to do more work before we can really demonstrate this
```

Comment:
I'd leave these example lines out until we have examples to show. Otherwise this will be included in the generated docs.

Reply:
Got it. Ripping these out.